{{ cookiecutter.project_slug }}/flows/query.yml

```diff
@@ -22,6 +22,3 @@ pods:
   ranker:
```

Comment:
in this example, I think we should not have a Ranker, it is missleading since this Ranker under current jina state will do nothing

Reply:
Removed Ranker